README.md

```diff
@@ -128,21 +128,21 @@ For values, please use the values associated with the Discovery service you crea
 
 > Note: Make sure to enclose your values in double quotes.
 
-For the `input` value [2], use a default question such as "how do I turn on the heater?". 
+For the `input` value [2], use a default question such as "how do I turn on the heater?".
 
-Next, go to the `Endpoints` panel [1]:
+Now that the credentials are set, along with our test question, return to the `Code` panel and press the `Invoke` button again. Now you should see actual results returned from the Discovery service:
 
-![](doc/source/images/action-endpoint.png)
+![](doc/source/images/action-code-invoke.png)
```

Comment:
We like a default value in the square brackets:
[action_code_invoke](doc/source/....)